npm-management.md

```diff
@@ -37,6 +37,12 @@ well.
 The credentials required for the `nodejs-foundation` user are maintained in
 encrypted form in the [secrets repo][].
 
+## Publishing new packages
+
+To publish a new npm package, send a request to nodejs/admin, pinging TSC and
```

Comment:
```suggestion
To publish a new "official" npm package for the first time from a repository under the `nodejs/` GitHub organization, send a request to nodejs/admin, pinging TSC and
```

honestly just suggesting this to get it wrapped up - we probably don't need to continue having this open, it's pretty good and can be further modified as the need arises.

Reply:
No scare-quotes around _official_:

```
To publish a new official npm package for the first time from a repository under the `nodejs/` GitHub organization, send a request to nodejs/admin, pinging TSC and
```

Reply:
And I agree this can land as-is and be iterated on.

Reply:
```suggestion
To publish a new npm package, send a request to nodejs/admin, pinging TSC.
```